### ISSUE TITLE
Sign language videos don't appear on iOS

### ISSUE THREAD

**Issue by colinbdclark** [accessibility-in-action/platform#1975]
**Prerequisites**

- [x] I've [searched](https://github.com/accessibility-exchange/platform/issues) for existing issues

**Describe the bug**
On the staging site, ASL videos aren't visible when viewed on iOS with Safari.

**To Reproduce**
1. Go to [the staging site](https://staging.accessibilityexchange.ca) on iOS
2. Tap one of the 'Sign Language Video' buttons (e.g. underneath the "What is the Accessibility Exchange?" header)
3. You'll see a purple horizontal bar show up, but the video is not visible.

**Expected behavior**
ASL videos should be visible and usable on iOS.

**Screenshots**
![IMG_803339CBBBEE-1](https://github.com/accessibility-exchange/platform/assets/558076/6daa9a68-7468-4a9b-b2b6-1369a0a9ea9f)

**Details**
- Device: iPhone XR
- Operating system: 17.03
- Browser: Safari
- Link to affected page: https://staging.accessibilityexchange.ca

**Additional context**
Logan also tested this on his phone, an iPhone 12 running iOS 16.6.1 using the Chrome browser and found the same issue.

**Comment by jobara** [comment]
I believe this is actually an issue with URLs for the video having changed. Some of the old videos weren't marked as unlisted and when this was updated in Vimeo the URLs change. I was able to reproduce the issue in Firefox on macOS by going to the staging site. Looking at a the URL used for the video I can see it doesn't include the extra info that an unlisted video would.

This has been corrected already and pushed to dev but changes haven't been deployed to the staging site yet. It will be updated soon as we move to the friends and family release.
